### ISSUE TITLE
[Feature] Add additional sort options to whitelist.

### ISSUE THREAD

**Issue by Cvrt19** [stashapp/stash#4967]
A recent update (https://github.com/stashapp/stash/pull/4865) enforced a whitelist for sort values, but this broke some unofficial functionality that I had been using. Before, I could add things like sortby=career_length or sortby=weight, or really any of the already available filtering options, and it would sort appropriately despite not being default selectable sort options. In particular, I had been using sorting by career_length to display Performers by date of debut and it was working perfectly, but now it's no longer supported because it's not on the official whitelist.

To sum up, my request is to add more of the existing filtering options to the sort whitelist since they're already available and working. Thanks very much.
